include/dem/particle_particle_contact_info_struct.h

```diff
@@ -24,15 +24,26 @@
 #  define particle_particle_contact_info_struct_h
 
 /**
- * This struct handles the information related to the calculation of the
+ * This class handles the information related to the calculation of the
  * particle-particle contact force
  */
 
 using namespace dealii;
 
 template <int dim>
-struct particle_particle_contact_info_struct
+class particle_particle_contact_info
```

Comment:
Would it be a good idea to add information of the particles (overlap, properties, and on..) that is destroyed after their usage (but not the tangential torque)? Just a question of readability and reducing the number of arguments.

Reply:
Good question, I am not sure about it but it is something to investigate.